src/main/solution_space/path_finder_test_system.py

```diff
@@ -130,11 +132,12 @@ def get_result_table(self, title: str) -> Optional[PrettyTable]:
         table = PrettyTable(field_names=[e.value for e in TEST_INPUT] +
                                         [self.__get_path_finder_version(pf) for pf in path_finders], title=title)
 
-        for test_input in self._test_inputs:
+        for i, test_input in enumerate(self._test_inputs):
             user_anon_tree, user_canon_tree = self.__create_user_trees(test_input)
+            test_input[TEST_INPUT.INDEX] = i
```

Comment:
Just add `i` in the row instead of putting INDEX in the `test_input` dict, the latter is more complicated and hard to understand